### DISCUSSION TITLE
How is this setup to autostart when isntalled via package manger?

### DISCUSSION THREAD

**Discussion by Juts2** [foresto/joystickwake#15] [Q&A] [answered]
I'm trying to figure out what method this is using to autostart on login. I've installed it from the AUR and I can see it was called by systemd but dont see it in any of the normal startup timers / locations.

**Comment by foresto** [comment, marked as answer]
Most desktop environments launch applications that have a .desktop file in the /etc/xdg/autostart directory. Joystickwake provides a .desktop file for this purpose. The AUR package you installed presumably includes the file and places it accordingly.

https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html

https://wiki.archlinux.org/title/XDG_Autostart

**Comment by Juts2** [comment]
Thank you! I was looking for a .service file, or a startup file in /home/user/.config/autostart. 

Seems like there are a dozen ways to do this.
